docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -380,15 +380,26 @@ V&V Checks:
     - XX relative risk on incidence of encephalopathy if birthing parent experiences obstructed labor
     - Need to determine how neonatal mortality in general will be modeled and how we will handle overlaps with preterm and LBWSG RR's on all cause mortality
   * - 7
-    - XX% of each type of facility have CPAP or NICU capabilities
-    - 
+    - Percentage of each facility type that have CPAP capabilities: CMONC - 39.3% and BMONC - 7.5% 
+    - These values are from the 2016 EmONC Final Report and are therefore only reflective of Ethiopian health systems. Please use these as a placeholder for now while we 
+      try to find reliable values for Nigeria and Pakistan. 
   * - 8
-    - XX relative risk on RDS mortality of facility having CPAP or NICU capabilities vs. not
-    - Need to confirm this will impact mortality not incidence. Also need to determine how neonatal mortality in general will be modeled and how we will handle overlaps with preterm and LBWSG RR's on all cause mortality
+    - 0.64 (95% CI 0.50-0.82) relative risk on RDS mortality of facility having CPAP capabilities 
+    - Need to confirm this will impact mortality not incidence. Also need to determine how neonatal mortality in general will be modeled 
+      and how we will handle overlaps with preterm and LBWSG RR's on all cause mortality. Note that we might want RR for NICU, but this value 
+      is for CPAP capabilities. Source: `2020 Cochrane review <https://pmc.ncbi.nlm.nih.gov/articles/PMC8094155/>`_
   * - 9
-    - XX relative risk for RDS incidence based on birthing parent receiving antenatal corticosteroids
-    - Need to confirm this will impact incidence not mortality. Also need to determine how neonatal mortality in general will be modeled and how we will handle overlaps with preterm and LBWSG RR's on all cause mortality
-
+    - 0.66 (95% CI 0.56-0.77) relative risk for RDS incidence based on birthing parent receiving antenatal corticosteroids
+    - This value is for incidence of RDS, however there is also an RR on RDS mortality (0.69, 95% CI 0.59-0.81). Also need to determine how neonatal mortality 
```

Comment:
We probably want 0.69 (0.59-0.81) as our RR, since we are only modeling mortality.  It is reassuring that these values are similar.  At this point, we have a strategy for how to model NN mortality in general and how to handle the overlaps.

Reply:
We can delete the sentence "Also need to determine how neonatal mortality in general will be modeled and how we will handle overlaps with preterm and LBWSG RR's on all cause mortality," because we've figured that out.

Also, it would be helpful to note who the intervention is given to, to make sure we model the intervention correctly. The Cochrane review says:
> Types of participants
Women, with a singleton or multiple pregnancy, expected to deliver preterm as a result of either spontaneous preterm labour, preterm prelabour rupture of the membranes or planned preterm delivery.